src/script/lua_api/l_camera.h

```diff
@@ -0,0 +1,47 @@
+#ifndef L_CAMERA_H
+#define L_CAMERA_H
+
+#include "l_base.h"
+#include "camera.h"
```

Comment:
please remove include , class has been declared, and include camera.h in cpp file